Example/SnapshotTests/BPKSnapshotTest.h

```diff
@@ -1,6 +1,7 @@
 /*
  *  Copyright (c) 2017-2018, Uber Technologies, Inc.
  *  Copyright (c) 2015-2018, Facebook, Inc.
+ *  Copyright (c) 2018-2021, Skyscanner Ltd.
```

Comment:
@georgegillams why doesn't this file have our typical header? Did we copy some of this code from `FBSnapshotTestCase`?